app/routes/_index.tsx

```diff
@@ -3,6 +3,7 @@ import { useState } from "react";
 
 import { Button } from "~/components/ui/button";
 import CardPlatform from "~/components/ui/card/card-platform";
+import DeleteMemberModal from "~/components/ui/modals/DeleteMemberModal";
 import OtpAuth from "~/components/ui/otp/OtpAuth";
 import { Input } from "~/types/otpauth";
 
```

Comment:
Please look for the person using your component on their page, and tell them to integrate it there. This route isn't where you do that.

Reply:
@chumex412 i have fixed the issues you talked about